Editor/Window/ConnectToFleetInput.cs

```diff
@@ -0,0 +1,205 @@
+            {
+                await UpdateFleetMenu();
+                _fleetState = FleetStatus.Creating;
+            }
+            
+            UpdateGUI();
+        }
+
+        private void OnCancelButtonClicked()
+        {
+            if (_fleetState is FleetStatus.Creating)
+            {
+                _fleetState = FleetStatus.Selected;
+            }
+            
+            UpdateGUI();
+        }
+        
+        private void OnSelectFleetDropdown()
+        {
+            _fleetState = FleetStatus.Selected;
+            
+            UpdateGUI();
+        }
+
+        private void RegisterCallBacks(VisualElement container)
+        {
+            container.Q<Button>("AnywherePageCreateFleetButton").RegisterCallback<ClickEvent>(async _ => 
+                await OnAnywhereConnectClicked(_fleetNameInput.text));
+            container.Q<Button>("AnywherePageConnectFleetNewButton").RegisterCallback<ClickEvent>(async _ => 
+                await OnCreateNewFleetClicked());
+            _fleetNameDropdownContainer.RegisterValueChangedCallback(evt => OnSelectFleetDropdown());
+            _cancelButton.RegisterCallback<ClickEvent>(_ => OnCancelButtonClicked());
+        }
+
+        private void AssignUiElements(VisualElement container)
+        {
+            _fleetNameInput = container.Q<TextField>("AnywherePageCreateFleetInput");
+            _fleetNameDropdownContainer = container.Q<DropdownField>("AnywherePageConnectFleetDropdown");
+            _fleetId = container.Q("AnywherePageConnectFleetID");
+            _fleetIdText = container.Q<Label>("AnywherePageConnectFleetIDDisplay");
+            _fleetStatus = container.Q("AnywherePageConnectFleetStatus");
+            _fleetCreateFoldout = container.Q("AnywherePageCreateFleetTitle");
+            _fleetConnectFoldout = container.Q("AnywherePageConnectFleetTitle");
+            _cancelButton = container.Q<Button>("AnywherePageCreateFleetCancelButton");
+        }
+        
+        private async Task SetupFleetMenu()
+        {
+            await UpdateFleetMenu();
+            _fleetNameDropdownContainer.RegisterValueChangedCallback(_ =>
+                {
+                    _stateManager.SelectedFleetName = _fleetNameDropdownContainer.value;
+                    var currentFleet = _fleetsList.First(fleet => fleet.Name == _fleetNameDropdownContainer.value);
+                    _fleetIdText.text = currentFleet.FleetId;
+                    FleetId = currentFleet.FleetId;
+                    _stateManager.CoreApi.PutSetting(SettingsKeys.SelectedFleetName, currentFleet.Name);
+                }
+            );
+        }
+        
+        private async Task UpdateFleetMenu()
+        {
+            if (_stateManager.GameLiftWrapper != null)
+            {
+                _fleetsList = await _fleetManager.ListFleetAttributes();
+                s_fleetNameList.Clear();
+                _fleetsList.ForEach(fleet => s_fleetNameList.Add(fleet.Name));
```

Comment:
We should be filtering to only the Anywhere fleets

Reply:
Added filter